content/en/_index.md

```diff
@@ -146,6 +165,14 @@ public static void main(String[] args) {
     exampleSnippet(projectId, filePath);
 }
 {{< /tab >}}
+{{< tab header="Go" >}}
+func main() {
```

Comment:
Go samples have typically lived in non-main packages. For this to work, we'll have to add a new package for every set of samples. Is that the expectation?

Reply:
This was made java specific in #41 until we can reach a broader consensus on how to handle this in each language. You can remove it from this pr.